app/src/main/res/layout/include_omnibar_toolbar.xml

```diff
@@ -105,6 +105,17 @@
                             android:padding="4dp"
                             android:src="@drawable/ic_loupe_24dp" />
 
+                        <FrameLayout
```

Comment:
`cookie_scene_1 ` is already a FL, why not merging the two?

Reply:
I don't know all details here since this belongs to the cookie banner animation (and that's already in `develop`, here I'm just integrating it with the privacy shield). 

Probably you are right and we could remove one level, but I would prefer to avoid changing the structure in this PR.